packages/app-builder-lib/src/util/appFileCopier.ts

```diff
@@ -188,32 +188,40 @@ export async function computeNodeModuleFileSets(platformPackager: PlatformPackag
   const result = new Array<ResolvedFileSet>()
   let index = 0
   const NODE_MODULES = "node_modules"
-  const getRealSource = (source: string) => {
-    const parentDir = path.dirname(source)
+  const getRealSource = (name: string, source: string) => {
+    const normalizedPath = source.split(path.sep).join("/")
```

Comment:
I think we can use `path.normalize` here or `path.resolve`. If it needs to be `"/"`, then we can `import * as pathPosix from "path/posix"` IIRC. What are your thoughts?

Reply:
This approach is indeed a bit confusing. I've modified it by changing name(@electron/remote) to `@electron + path.sep + remote`, and then truncating the package name from the suffix in the source. This should work.